components/Toast/toast.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Toast</title>
+    <link rel="stylesheet" href="../../styles/main.css" />
+  </head>
+  <body>
+    <!-- success toast -->
+    <div
+      id="toast-success"
+      class="toast text-sm d-flex align-center p-3 rounded-sm d-none"
+    >
+      Added Successfully !
+      <button type="button" class="btn btn-icon">
+        <i class="fas fa-times text-md btn-toast-close ml-4"></i>
+      </button>
+    </div>
+    <button
+      id="success-toast-demo-btn"
+      type="button"
+      class="btn btn-primary rounded-sm text-sm p-3"
+    >
+      Success Toast Demo
+    </button>
+
+    <!-- error toast -->
+    <div
+      id="toast-error"
+      class="toast text-sm d-flex align-center p-3 rounded-sm d-none"
+    >
+      Try Again Later !
+      <button type="button" class="btn btn-icon">
+        <i class="fas fa-times text-md btn-toast-close ml-4"></i>
```

Comment:
Nothing major but for good UI you can decrease the cross size here, cross is looking a little too big

Reply:
I think it should be big , otherwise user will have problem while using that button